dll/win32/syssetup/install.c

```diff
@@ -567,6 +567,9 @@ ShowStatusMessageThread(
 
     ShowWindow(hWnd, SW_SHOW);
 
+    hItem = GetDlgItem(hWnd, IDC_STATUSPROGRESS);
+    if (hItem) PostMessage(hItem, PBM_SETMARQUEE, TRUE, 40);
+
```

Comment:
Is the 571 line supposed to be this way? Shouldn't the conditional code be wrapped with curly braces? Like in this image:
https://imgur.com/a/KuWmp

Reply:
Don't know whether it's required here or not, but since there is only one operation, it works.

Reply:
This works, although it's preferred to use a separate line (for step-through) and to always use braces.

Reply:
Allright then :)